test/lib/testMayaUsdProxyAccessor.py

```diff
@@ -0,0 +1,1265 @@
+        return False
+
+    @staticmethod
+    def waitForCache(unit_test, wait_time=5):
+        '''
+        Fill the cache in the background, waiting for a maximum time
+        :param unit_test: The test object from which this method was called
+        :param wait_time: Time the test is willing to wait for cache completion (in seconds)
+        '''
+        cmds.currentTime( cmds.currentTime(q=True) )
+        cmds.currentTime( cmds.currentTime(q=True) )
+        cache_is_ready = cmds.cacheEvaluator( waitForCache=wait_time )
+        unit_test.assertTrue( cache_is_ready )
+
+    @staticmethod
+    def is_caching_scope():
+        '''
+        Method to determine whether caching is on or off in this object's scope
+        :return: True, since this is the caching scope
+        '''
+        return True
+
+def isPluginLoaded(pluginName):
+    """
+    Verifies that the given plugin is loaded
+    Args:
+        pluginName (str): The plugin name to verify
+    Returns:
+        True if the plugin is loaded. False if a plugin failed to load
+    """
+    return cmds.pluginInfo( pluginName, loaded=True, query=True)
+
+def loadPlugin(pluginName):
+    """
+    Load all given plugins created or needed by maya-ufe-plugin
+    Args:
+        pluginName (str): The plugin name to load
+    Returns:
+        True if all plugins are loaded. False if a plugin failed to load
+    """
+    try:
+        if not isPluginLoaded(pluginName):
+            cmds.loadPlugin( pluginName, quiet = True )
+        return True
+    except:
+        print(sys.exc_info()[1])
+        print("Unable to load %s" % pluginName)
+        return False
+
+def isMayaUsdPluginLoaded():
+    """
+    Load plugins needed by tests.
+    Returns:
+        True if plugins loaded successfully. False if a plugin failed to load
+    """
+    # Load the mayaUsdPlugin first.
+    if not loadPlugin("mayaUsdPlugin"):
+        return False
+
+    # Load the UFE support plugin, for ufeSelectCmd support.  If this plugin
+    # isn't included in the distribution of Maya (e.g. Maya 2019 or 2020), use
+    # fallback test plugin.
+    if not (loadPlugin("ufeSupport") or loadPlugin("ufeTestCmdsPlugin")):
+        return False
+
+    # The renderSetup Python plugin registers a file new callback to Maya.  On
+    # test application exit (in TbaseApp::cleanUp()), a file new is done and
+    # thus the file new callback is invoked.  Unfortunately, this occurs after
+    # the Python interpreter has been finalized, which causes a crash.  Since
+    # renderSetup is not needed for mayaUsd tests, unload it.
+    rs = 'renderSetup'
+    if cmds.pluginInfo(rs, q=True, loaded=True):
+        unloaded = cmds.unloadPlugin(rs)
+        return (unloaded[0] == rs)
+
+    return True
+
+def makeUfePath(dagPath, sdfPath=None):
+    """
+    Make ufe item out of dag path and sdfpath
+    """
+    ufePath = ufe.PathString.path('|world{},{}'.format(dagPath,sdfPath) if sdfPath != None else '|world{}'.format(dagPath))
```

Comment:
"|world" prefix is optional when creating a path using Ufe::PathString::path(), and should preferably be omitted.

Reply:
Looks to be working, thx! 90eccd8f